packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/mask.tests.js

```diff
@@ -520,7 +521,8 @@ QUnit.module('format: fixed point format', moduleConfig, () => {
 
     QUnit.test('value option should have right value after inserting when format is enabled and decimalSeparator is \',\' (T829935)', function(assert) {
         const oldDecimalSeparator = config().decimalSeparator;
-        config({ decimalSeparator: ',' });
+
+        executeActionWithSuppressErrors(() => config({ decimalSeparator: ',' }));
```

Comment:
I suggest u to:
1. Move all these tests to the single module `"," as a decimal separator`
2. Set a necessary locale using `,` as a decimal separator instead of the current hacky approach with a config